README.md

```diff
@@ -76,7 +76,7 @@ st.use(userRuntype, {id: 1, name: 'matt', isAdmin: true})
 // => {ok: false, error: FAIL}
 
 st.getFormattedError(FAIL)
-// => 'invalid keys in record ["isAdmin"] at `<value>` in `{"id":1,"name": "matt", ... }`'
+// => 'invalid keys in record: ["isAdmin"] at `<value>` in `{"id":1,"name": "matt", ... }`'
```

Comment:
There are other fixes that need to be made to the README to bring it up to date with the current version of `simple-runtypes`, but I have left those for now to prevent scope creep. Thise fixes can be done in a separate PR.